ci-operator/step-registry/openshift-qe/network-perf/openshift-qe-network-perf-ref.yaml

```diff
@@ -5,6 +5,10 @@ ref:
     name: ocp-qe-perfscale-ci
     tag: latest
   commands: openshift-qe-network-perf-commands.sh
+  env:
+  - name: REGION
```

Comment:
This variable has no application in our tests. I dont think it should be added here